packages/attendance_management/lib/pages/mark_attendance.dart

```diff
@@ -272,42 +264,48 @@ class _MarkAttendancePageState extends State<MarkAttendancePage> {
                                 ),
                               ),
                               DigitCard(
-                                child: Column(
-                                  mainAxisAlignment: MainAxisAlignment.start,
+                                  margin:
+                                      EdgeInsets.all(theme.spacerTheme.spacer3),
                                   children: [
-                                    Padding(
-                                      padding:
-                                          const EdgeInsets.only(bottom: 8.0),
-                                      child: tableData.isNotEmpty
-                                          ? DigitTable(
-                                              height: tableData.length > 2
-                                                  ? (tableData.length + 1) * 57
-                                                  : (tableData.length + 1) * 65,
-                                              headerList: headerList(
-                                                widget.dateTime,
-                                                localizations,
-                                              ),
-                                              tableData: tableData,
-                                              columnWidth: 140,
-                                              scrollPhysics:
-                                                  const NeverScrollableScrollPhysics(),
-                                            )
-                                          : NoResultCard(
-                                              align: Alignment.center,
-                                              label: localizations.translate(
-                                                i18.common.noResultsFound,
-                                              ),
+                                    tableData.isNotEmpty
+                                        ? table.DigitTable(
+                                            tableHeight: viewOnly
+                                                ? MediaQuery.of(context)
+                                                        .size
+                                                        .height *
+                                                    .55
+                                                : MediaQuery.of(context)
+                                                        .size
+                                                        .height *
+                                                    .4,
+                                            tableWidth: MediaQuery.of(context)
+                                                .size
+                                                .width,
+                                            stickyHeader: false,
```

Comment:
If default value is the given value remove those fields

Reply:
Removed stickyHeader